blurb/blurb.py

```diff
@@ -701,7 +701,7 @@ def run(s):
     return process.stdout.decode('ascii')
 
 
-readme_re = re.compile(r"This is Python version [23]\.\d").match
+readme_re = re.compile(r"This is \w+thon version [23]\.\d").match
```

Comment:
```suggestion
readme_re = re.compile(r"This is \w+ version \d\.\d+").match
```
What about relaxing it a bit more to prevent bikeshedding on the exact names to be matched?
This should still fulfill its intended purpose for CPython and might allow other forks to use blurb, with the understanding that it remains CPython-specific and there are no explicit guarantees that it will work with other repos.

Reply:
I agree. Do we really care what name is used here? For anyone arguing for strictness: what error is this trying to prevent?